src/content.js

```diff
@@ -452,6 +452,24 @@ function addTitleToEmojis() {
 	}
 }
 
+function overrideCodeFonts() {
+	$(`
+	code,
+	kbd,
+	pre,
+	tt,
+	.CodeMirror-lines,
+	.ace_editor.ace-github-light,
+	.blob-code-inner,
+	.export-phrase pre,
+	.file-editor-textarea,
+	.gollum-editor .expanded textarea,
+	.gollum-editor .gollum-editor-body,
+	.input-monospace,
+	.wiki-wrapper .wiki-history .commit-meta code`)
+	.addClass('override-fonts');
+}
+
```

Comment:
That's a lot of operations. I'd
1. leave this in the CSS
2. prefix each selector with `.override-fonts`
3. Only add the `override-fonts` class to `<html>` via JS